transport/network/src/ping.rs

```diff
@@ -1,15 +1,12 @@
-use std::{
-    collections::{hash_map::Entry, HashMap},
-    ops::Div,
-};
+use std::ops::Div;
```

Comment:
I think the entire ping module should receive more documentation, especially given the refactor.